inference-engine/src/cldnn_engine/ops/constant.cpp

```diff
@@ -103,7 +103,7 @@ static void CreateConstantOp(Program& p, const std::shared_ptr<ngraph::op::v0::C
                     bool all_inputs_1d = true;
                     for (size_t j = 0; j < outOp->get_input_size(); j++) {
                         auto& in_shape = outOp->get_input_shape(j);
-                        if (in_shape.size() != 1)
+                        if (in_shape.size() > 1)
```

Comment:
Could you add new test case into inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/eltwise.cpp that covers this scenario?